webapp/src/lib/components/spot-listings/detail-modal.svelte

```diff
@@ -0,0 +1,54 @@
+<script lang="ts">
+    import { Modal } from 'carbon-components-svelte';
+    export let listing;
```

Comment:
We do have a parking spot type.
```suggestion
let type ParkingSpot = components["schema"]["ParkingSpot"]
    export let listing : ParkingSpot;
```

Reply:
I couldn't find it in the defined types

Reply:
it is in lib/sdk/schema.d.ts